Automattic/jetpack
# Subscriptions: Make Subscription overlay dismiss cookie a session cookie

Fixes #

## Proposed changes:

It makes a Subscription overlay dismiss cookie a session cookie.

<img width="967" alt="Screenshot 2024-08-27 at 13 14 50" src="https://github.com/user-attachments/assets/554d072a-71df-4fa4-ab75-15768b6bc621">

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Enable Subscription overlay in the Newsletter settings
* Visit the home page of your site
* Make sure the dismiss cookie is set as a session cookie
